src/operation/list_collections/mod.rs

```diff
@@ -76,6 +76,7 @@ impl Operation for ListCollections {
             description.server_address.clone(),
             self.options.as_ref().and_then(|opts| opts.batch_size),
             None,
+            None,
```

Comment:
Should the comment be propagated from somewhere in the server response?

Reply:
the bottom of [this section](https://github.com/mongodb/specifications/blob/master/source/enumerate-collections.rst#post-mongodb-280-rc3-versions) indicates that the comment should not be attached to `getMore`s (which what propagating the comment here would do)